docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -181,3 +183,27 @@ Deleting a file storage at an instance level deletes the Nextcloud integration c
 ## Using the integration
 
 Once the file storage is added and enabled for projects, your users are able to take full advantage of the integration between Nextcloud and OpenProject. For more information on how to link Nextcloud files to work packages in OpenProject and access linked work packages in Nextcloud, please refer to [Using the Nextcloud integration](../../../user-guide/nextcloud-integration/).
+
+
+
+## Troubleshooting
+
+On Nextcloud inside the OpenProject integration App when adding the OpenProject host it shows the error **"Please enter a valid OpenProject hostname"**.
+
+- If you are on a local network with your Nextcloud and OpenProject hosts, it might be necessary to explicitly allow local remote servers by setting a system configuration flag on your Nextcloud host's command line. Use the command `sudo -u www-data php occ config:system:set allow_local_remote_servers --value 1` in order to change the setting for Nextcloud.
+
+- To test the connection between OpenProject and the Nextcloud and the OpenProject integration App, you are able to use the following methods from your OpenProject host's command line:
+
+  ```bash
+  # Check Connection from OpenProject to Nextcloud
+  curl --location --request GET 'https://subdomain.domain.tld/index.php/apps/integration_openproject/check-config' --header 'Authorization: foo'
+  # If Nextcloud is integrated correctly the response should look like the following
+  # {"user_id":"","authorization_header":foo}
+  ```
+
+  ```bash
+  # Check Connection from OpenProject to Nextcloud OpenProject integration App
+  curl -H 'OCS-APIRequest:true' -H 'Accept:application/json'https://subdomain.domain.tld/nextcloud/ocs/v2.php/cloud/capabilities
```

Comment:
What does this test? And how can it help to solve a problem?

Reply:
it comes from you :)

Reply:
Sure, it comes from me and was part of some debugging with a client I guess. But you decided to put it here in the docs, which makes sense as that can help debugging Network issues(!). So you might want to explain the reader why/when this check comes in handy ("Cannot setup connection with Nextcloud server"). Explain what the test does ("It basically does what the validation of the input field "Nextcloud host" also does. It could give you more additional information of what actually goes wrong in your setup.") Also, the reader needs to get some help with how to interpret the test. What are expected results? ("Returned response should have the HTTP status code 200 "OK". The Nextcloud version should be at least 22.") And what are bad test results? (Name some examples: Error codes such as "Not found", DNS resolution issues, SSL encryption issues,.. ) What can the reader do next in case of a bad test results? ("Verify that the URL is correct. If it is correct contact your network administrator to further investigate that issue.")

Or to put it in other words: Put yourself in the shoes of the reader that has an issue and check if you understand the docs here. The reader needs:

1. A description of the problem to understand that it is her problem. 
2. Then there needs to be the test/check if that is actually the problem with detailed step by step explanation of how to conduct it.
3. Then the interpretation with examples of positive and negative test results.
5. Then the remedy/help. (Do this, check that, read up these docs (link) etc.)
6. Sometimes it needs a verification that the remedy helped. Most likely the same as step 2.

I understand that you don't know all the nitty gritty details of the integration. However, verifying the completeness of that pattern above will at least create "Todos" for clarification with me, @apfohl, or @Kharonus.